src/SystemSettings.cpp

```diff
@@ -15,7 +15,7 @@ SystemSettings::SetDefaults()
   } else {
     devices[0].port_type = DeviceConfig::PortType::SERIAL;
 #ifdef _WIN32
-    devices[0].path = _T("COM1:");
+    devices[0].path = _T("\\\\.\\COM1");
```

Comment:
Your code is now inconsistent regarding what `path` means. Is this suposed to be a device name? Device name with colon? UNC path? You should document that, have proper migration for old profile values, and make your code consistent.